docs/architecture/deployment/installation-guide.md

```diff
@@ -36,38 +36,105 @@ To install Prophecy, the following infrastructure requirements must be satisfied
+| `global.prophecy.wildcardCert`             | -                        |                                                                                                                                                                                                                                  |                                     |
+| `global.prophecy.wildcardCert.name`        | string                   | List of comma separated secret cert names.                                                                                                                                                                                       | `""`                                |
+| `global.prophecy.wildcardCert.useExternal` | bool                     | Setting this to true will require the customer to provide a secret name here.                                                                                                                                                    | `false`                             |
+| `global.repository`                        | string                   | Container registry prefix to be used.                                                                                                                                                                                            | `gcr.io/prophecy-share`             |
+| `global.tls`                               | -                        |                                                                                                                                                                                                                                  |                                     |
+| `global.tls.enabled`                       | bool                     | Specifies if TLS is to be enabled                                                                                                                                                                                                | `true`                              |
+| `global.tls.certOnLB`                      | bool                     | Specifies wheather TLS termination is to be done at the loadbalancer                                                                                                                                                             | `false`                             |
+| **POSTGRES**                               |
+| `postgres.isExternalPostgres`              | bool                     | Setting this to true, will allow Prophecy applications to use a externally managed Postgres instance and will not use a Prophecy managed instance.                                                                               | `false`                             |
```

Comment:
*will allow Prophecy services to use an externally managed

Reply:
```suggestion
| `postgres.isExternalPostgres`              | bool                     | Setting this to true, will allow Prophecy services to use a externally managed Postgres instance and will not use a Prophecy managed instance.                                                                               | `false`                             |
```